CONFIGURATION.md

```diff
@@ -153,6 +153,8 @@ tls_config:
 
 [ transport_protocol: <string> | default = "udp" ] # udp, tcp
 
+[ query_class: <string> | default = "IN" ]
```

Comment:
I'd put this below the type, as it'll be less common.

Reply:
Moved :)